jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -139,181 +111,259 @@ public static String printSmartLocators(IBaseElement el) {
                     return null;
                 break;
         }
-        String locatorName = ELEMENT.smartName.value.execute(el.getName());
+        String locatorName = jdiSettings.ELEMENT.smartName.value.execute(el.getName());
         return el.base().timer().getResult(() -> {
-            String locator = format(ELEMENT.smartTemplate, locatorName);
-            UIElement ui = (ELEMENT.smartTemplate.equals("#%s")
-                ? $(locator)
-                : $(locator, el.base().parent))
+            String locator = format(jdiSettings.ELEMENT.smartTemplate, locatorName);
+            UIElement ui = (jdiSettings.ELEMENT.smartTemplate.equals("#%s")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)